docs/platforms.openstack.md

```diff
@@ -23,3 +23,7 @@ meshcloud allows creation of [Service Users](meshcloud.service-user.md) for Open
 ## Quota Management
 
 Partners can enforce detailed per-project quotas for OpenStack via meshcloud.
+
+## Federated Idp
+
+meshIdB is not required for the integration anymore. You can directly integrate your company-wide IdP to OpenStack. meshStack will make sure that users have access to the OpenStack projects they are assigned to in meshStack.
```

Comment:
d: Perhaps this has to be moved to the OpenStack Platform Integration page in the operators section of the docs? Or is it that already? I think we have to explain a bit more in here. So stuff like that we create users if they don't exist yet, that we set the federated attribute accordingly. Link the OpenStack docs that explain this. Describe that we currently create one group per project and assign the users to that group.

Reply:
Yes. This makes sense. We should move this part to `meshtack.openstack.index.md`.
 I've extended the description a little bit.